pkg/reporting/nodes_test.go

```diff
@@ -1,30 +1,100 @@
-//
-// Copyright 2019 Chef Software, Inc.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-//
-
 package reporting_test
 
 import (
+	"encoding/json"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
+	"fmt"
 
-	subject "github.com/chef/chef-analyze/pkg/reporting"
```

Comment:
Lets talk about this `subject` import pattern, also the best practice is to sort the imports in this order:
```go
import (
    // Go native libraries like:
    "fmt"

    // Empty line that divides the next section, Community Go packages like:
    chef "github.com/chef/go-chef"
    "github.com/stretchr/testify/assert"

    // Another empty line to separate the next section, internal packages:
    subject "github.com/chef/chef-analyze/pkg/reporting"
)
```

Reply:
I think goimport removed that import because I wasn't referencing subject.  I'll make that change if that's the preferred pattern.